### ISSUE TITLE
Issue event object with event `labeled` doesn't have label attributes

### ISSUE THREAD

**Issue by kiran-vemuri** [PyGithub/PyGithub#653]
I am trying to access the label name from a `labeled` event and looks like that attribute is missing.

```
"label": {
            "name": "sample-label",
            "color": "fbca04"
        }
```

I can get this data using a REST call. Can we add that?

**Comment by sfdye** [comment]
@kiran-vemuri Thanks for report. Can you elaborate a bit more and maybe provide a code snippet?

**Comment by allevin** [comment]
@kiran-vemuri, I believe that this issue is corrected with #857.  If so, can you close?

cc: @sfdye
